hibike/devices/Potentiometer/potentiometer.h

```diff
@@ -24,5 +21,6 @@ hibike_uid_t UID = {
 void setup();
```

Comment:
Please update the year to 1.  The Year is the number of years since Year 8, the 2015 - 2016 season.